docs/jade-getting-started.md

```diff
@@ -342,8 +342,6 @@ jade-data-repo/application.properties.  You can convert any application.property
 variable by switching to upper case and every "." to "_".
 
 ```
-export DATAREPO_USEREMAIL={your dev gmail account}
-
```

Comment:
This change is unrelated, correct?

Reply:
Yes, I happened to commit it, it's a change I had lying around from before. I can remove it, although it should be updated at some point.